### ISSUE TITLE
Maps - * A find my location button to easily find stuff near me

### ISSUE THREAD

**Issue by s3m1s0n1c** [DevLAN-io/RFSPager-issues#4]
* A find my location button to easily find stuff near me
Like on google maps etc with the big arrow thing

**Comment by nicko170** [comment]
@s3m1s0n1c  thoughts on alert zones within X km of location for notifications as well?  Might be a PITA to impliment, but I am game!

(like the FiresNearMe app)

**Comment by Loganrfs** [comment]
alert zones would be good with a X zone radius of locations so any incidents come in that zone the user would get a notification.

**Comment by s3m1s0n1c** [comment]
Yeah sounds good.. I was thinking more generally the way you had it..
Just to use Phone location and zoom in where u are..

But alert zones sounds like a good addition as well :)
Just set the range in settings.
